gr-digital/include/gnuradio/digital/glfsr.h

```diff
@@ -27,22 +27,22 @@ namespace digital {
 class DIGITAL_API glfsr
 {
 private:
-    uint32_t d_shift_register;
-    uint32_t d_mask;
+    boost::uint32_t d_shift_register;
```

Comment:
Thanks for this PR and noting that there's a potential issue with these types. Since we're using C++11 ... instead of moving these types into `boost::`, how about instead let's change `boost/cstdint.hpp` to `cstdint` and call this done without having to change any of the types herein? Less boost == better .. and, this would be a simpler solution.